cmd/tasks/src/lib.rs

```diff
@@ -228,11 +243,14 @@ fn tasks(
             // Always load R4, R5 and R6, which are in the saved state in our
             // task structure (and are needed to print state).
             //
-            for r in 4..=6 {
-                let o = offs + r4 + (r - 4) * 4;
+            for r in 0..=2 {
```

Comment:
R4/R5/R6 (0..=2) doesn't really apply directly to RV.

Reply:
This block of code is to pull the first 3 syscalls registers, which on arm are R4,R5,R6.  I think the code is correct, just was an out of date comment.

Reply:
SG.